docs/source/setup_lora.rst

```diff
@@ -21,152 +21,151 @@ first be created at TTN (if one does not already exist).
-   network (e.g.: geiger_20200205).
--  **Description:**
+   network (e.g.: geiger_20220105).
+**Application name**  
+  Choose an arbitrary name for your application.
+**Description:**
    Any description of the application can be entered here.
--  **Application EUI:**
-   Remains empty, the number is generated by the TTN system.
--  **Handler registration:**
-   The pre-filled value (ttn-handler-eu) is already correct and remains.
 
-Now add the application with **Add application** in the lower right
-corner.
+Now create the application by clicking the **Create application** button.
 
 Create device
 ^^^^^^^^^^^^^
 
-Finally, the device has to be created. To do this, select the newly
-created application in the overview of applications (click). Then in the
-middle area at **DEVICES** start the creation of a new device via
-**register device**. The following fields must be filled in:
-
--  **Device ID:**
-   Any name for the device. It must be unique within the application
-   (e.g.: geiger_01) and consist only of lower-case letters.
--  **Device EUI:**
-   Click once on the symbol on the far left of the line, then the text
-   appears that this number is generated by the system. We don’t have to
-   enter anything else.
--  **App Key:**
-   No input necessary
--  **App EUI:**
-   Stays like this
-
-Click on **Register** in the lower right corner. Congratulations, the
-device is created.
-
-Modifying the LoRa parameters
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
--  After the registration was completed, the LoRa parameters can be transferred to the program.
--  They can be set up at the configuration site of the Geiger counter (see above).
--  Go through the configuration site until the settings of the LoRa  parameters are displayed. 
-   Type in the 3 parameters from the TTN console (**APPEUI, DEVEUI, APPKEY**). 
-   They can be found in your TTN account for each device (see above). 
-   The HEX values must be entered **without** spaces as they appear in the TTN.
-
-Example:
-| The TTN console reads
+On the next page you can now create a new device.  Press the button **+ Add end device**.
+Now select **Manually** from the menue directly under **Register end device**.
+Then the following fieds must be filled in:
+
+**Frequency plan**
+  Select **Europe 863-870 MHz (SF9) for RX2 - recommended)**.
+**LoRaWan version**
+  Select **MAC V1.02**.
+**Regional Parameters version**
+  Select **PHY V1.0.2 REV B**.
+
+For each of the next three entries select **Generate** or **Fill with zeros**. Please write down these three values, 
+you will need them later on.
+
+The last field (**End device ID**) will be filled in automatically.
+Now press **Register end device**.
+
+Enter the LoRa parameters into the MultiGeiger
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  After the registration was completed, the LoRa parameters need to be transferred to the MultiGeiger program.
+-  These can be entered via the configuration page of the MultiGeiger (see above).
```

Comment:
After the registration was completed, the LoRa parameters need to be entered into the form on the MultiGeiger configuration page (see above).